website/src/pages/_app.tsx

```diff
@@ -1,43 +1,56 @@
 import "../styles/globals.css";
 import "focus-visible";
 
-import type { AppProps } from "next/app";
+import { boolean } from "boolean";
+import type { AppContext, AppProps } from "next/app";
 import Head from "next/head";
 import { SessionProvider } from "next-auth/react";
 import { appWithTranslation, useTranslation } from "next-i18next";
 import React, { useEffect } from "react";
 import { FlagsProvider } from "react-feature-flags";
 import { getDefaultLayout, NextPageWithLayout } from "src/components/Layout";
 import flags from "src/flags";
+import { isChatEnabled } from "src/lib/chat_enabled";
 import { SWRConfig, SWRConfiguration } from "swr";
 
 import nextI18NextConfig from "../../next-i18next.config.js";
-import { Chakra, getServerSideProps } from "../styles/Chakra";
+import { Chakra } from "../styles/Chakra";
 
 type AppPropsWithLayout = AppProps & {
   Component: NextPageWithLayout;
+  env: typeof process.env;
+  cookie: string;
 };
 
 const swrConfig: SWRConfiguration = {
   revalidateOnFocus: false,
   revalidateOnMount: true,
 };
 
-function MyApp({ Component, pageProps: { session, cookies, ...pageProps } }: AppPropsWithLayout) {
+function MyApp({ Component, pageProps: { session, ...pageProps }, env, cookie }: AppPropsWithLayout) {
   const getLayout = Component.getLayout ?? getDefaultLayout;
   const page = getLayout(<Component {...pageProps} />);
   const { t, i18n } = useTranslation();
+
   const direction = i18n.dir();
   useEffect(() => {
     document.body.dir = direction;
   }, [direction]);
+
+  // expose env vars on the client
+  if (typeof window !== "undefined") {
+    for (const key in env) {
+      process.env[key] = env[key];
+    }
```

Comment:
could we maybe do it like this?
```ts
process.env = new Proxy(env, {
    get(env, key) {
        const value = env[key];
        if (!value) {
           throw new Error("Environment variable not set in _app.tsx")
        }
        return value;
    }
)
```

I think it is very easy to make this mistake, which is why we should log how to fix it..

I hope nothing else breaks, otherwise just `Object.assign(process.env, env)`